lib/js/components/buttons/IconButton/IconButton.consts.ts

```diff
@@ -16,11 +15,9 @@ export const ICON_BUTTON_TYPES = {
 
 export const ICON_BUTTON_COLOR_SCHEMES = {
 	ALL_IN_COLOR: 'all-in-color',
-	MINOR_ICON: 'minor-icon',
-	MINOR_LABEL: 'minor-label',
+	NEUTRAL_LABEL: 'neutral-label',
 } as const;
 
 export const ICON_BUTTON_COLORS = {
```

Comment:
`ICON_BUTTON_COLORS` vs `BUTTON_ICON_COLORS` what's the difference? Do we need to have 2 variables?